src/Microsoft.Health.Fhir.Core/Messages/Operation/EverythingOperationRequest.cs

```diff
@@ -0,0 +1,51 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using EnsureThat;
+using MediatR;
+using Microsoft.Health.Fhir.Core.Models;
+
+namespace Microsoft.Health.Fhir.Core.Messages.Operation
```

Comment:
>Operation [](http://example.com/codeflow?start=46&length=9)

I would create it's own namespace and folder for `Everything`

There is two types of files right now in this namespace.
One to handle requests to get information about non-defined spec operation (Reindex, Convert) and validate operation requests, which I put here by mistake.